classes/order/OrderDetail.php

```diff
@@ -726,8 +715,9 @@ protected function setDetailProductPrice(Order $order, Cart $cart, $product)
      * @param int $id_order_status
      * @param int $id_order_invoice
      * @param bool $use_taxes set to false if you don't want to use taxes
+     * @param int $id_warehouse [no longer used]
```

Comment:
```suggestion
```

If there is not anymore the parameter, the phpdoc must not be defined (else PHPStan will trigger on it).